### ISSUE TITLE
Provide option to configure custom character encoding for clipboard

### ISSUE THREAD

**Issue by gothka404** [gujjwal00/avnc#271]
After encountered [issues with GNOME Remote Desktop](https://github.com/gujjwal00/avnc/issues/256#issuecomment-2408467779) clipboard text, I noticed that x11vnc has difficulties with UTF-8/unicode handling as well.  Both pasting UTF-8 characters with Remmina or AVNC is outputted as `?` in remote session.

Since some VNC servers still seems to expect encodings other than UTF-8, what about providing an option to customize  encoding?

In this context, I observed another issue: If using the "Send text to server" AVNC input field, order of characters is switched up.
Example: "Hello world" from AVNC is pasted as "hdlrow olle" or "dlrow olleh" in remote session.
(If it turns out to be a different problem, I can post a separate issue.)

**Comment by gujjwal00** [comment]
> Since some VNC servers still seems to expect encodings other than UTF-8, what about providing an option to customize encoding?

I don't wan't to add an option because this is a fixable server bug. I have opened an issue in libvncserver repo: https://github.com/LibVNC/libvncserver/issues/638 to fix the root cause. In the meantime, please use an alternative server like TigerVNC. 


> In this context, I observed another issue: If using the "Send text to server" AVNC input field, order of characters is switched up.

I am unable to reproduce this. For me, characters are always sent in correct order. Please tell me a bit more, e.g. any specific app this happens with. If you can post a screenshot/screen-recording etc., that would also be helpful.

**Comment by gothka404** [comment]
> I don't wan't to add an option because this is a fixable server bug. I have opened an issue in libvncserver repo

Understandable, thanks much for opening an issue.

> I am unable to reproduce this.

Server in essence is Debian 12 + X11 (disabled Wayland) + `x11vnc` package. GNOME Remote Desktop is enabled in parallel. `x11vnc` is started as systemd user service like:
```sh
x11vnc -forever -noxdamage -repeat\
  -display $DISPLAY\
  -auth $XDG_RUNTIME_DIR/gdm/Xauthority\
  -rfbauth %h/.vnc/passwd
```

Client is Android 15 (though custom ROM) + AVNC from F-Droid store.

In case this does not help for reproduction: I will need some time to do a bit more testing of other client and server constellations, like vanilla Android and server alternatives like TigerVNC.

**Comment by gujjwal00** [comment]
My PR to fix the issue in LibVNCCLient was merged, and v0.9.15 has been released. When the library is updated in Debian, both servers will be fixed.

**Comment by gothka404** [comment]
Much appreciated @gujjwal00 , thank you!
